packages/odf/components/topology/sidebar/TopologySideBarContent.tsx

```diff
@@ -49,6 +61,10 @@ const filterFactory = (kind: string) => {
   const nodeFilter = (name: string) => (alert: Alert) =>
     alert?.labels?.node === name || commonFilter(name)(alert);
 
+  if (kind === StorageClusterModel.kind) {
+    return isStorageClusterAlert;
```

Comment:
Just for consistency, I would rename it as:
```suggestion
    return storageClusterAlertFilter;
```